site/content/zh-CN/docs/tasks/run/multikueue/mpijob.md

```diff
@@ -1,36 +1,35 @@
 ---
-title: "Run MPIJobs in Multi-Cluster"
+title: "在多集群环境中运行 MPIJob"
 linkTitle: "MPIJob"
 weight: 5
 date: 2024-10-25
-description: >
-  Run a MultiKueue scheduled MPIJobs.
+description: 在多集群环境中运行 MPIJob
 ---
 
-## Before you begin
+## 开始之前 {#before-you-begin}
 
-Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+请查阅 [MultiKueue 安装指南](/zh-CN/docs/tasks/manage/setup_multikueue) 了解如何正确设置 MultiKueue 集群。
```

Comment:
```suggestion
请查阅 [MultiKueue 安装指南](/zh-CN/docs/tasks/manage/setup_multikueue)了解如何正确设置 MultiKueue 集群。
```

Reply:
Because the first character on the left is English, and the character before it is Chinese, a space is left between Chinese and English. The character on the right is Chinese, and the character after it is also Chinese, so there is no need to leave a space